sql/2020/05_Media/meta_open_graph.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+# usage meta open graph
+
+# returns all the data we need from _almanac
+CREATE TEMPORARY FUNCTION get_almanac_info(almanac_string STRING)
```

Comment:
Nit but should we have better names for these to be clear that it's not all almanac_info? This applies throughout.

```suggestion
CREATE TEMPORARY FUNCTION get_almanac_meta_og_info(almanac_string STRING)
```